### ISSUE TITLE
First Write Example - series.set_machine("str") is not working

### ISSUE THREAD

**Issue by ghost** [openPMD/openPMD-api#794]
**Describe the bug**
When following the **First Write** example on openPMD (https://openpmd-api.readthedocs.io/en/0.12.0-alpha/usage/firstwrite.html ) via python / Jupyter
series.set_machine throws error (auto fill can't see the option) 

**To Reproduce**
Compile-able/executable code example to reproduce the problem:

Python:
```py
import openpmd_api as io
import numpy as np

series = io.Series("myOutput/data_001.h5", io.Access.create)

i = series.iterations[42]

series.set_author(
    "i Lomi <i.lomi@smth.de>")
series.set_machine(                      #This is not working
    "Some Machine")
series.set_attribute(
    "Foot", "Schnitzel")
i.set_attribute(
    "hot", True)
```

**Expected behavior**

AttributeError: 'openpmd_api.openpmd_api_cxx.Series' object has no attribute 'set_machine'

**Software Environment**
 - version of openPMD-api: [0.12.0-api]
 - installed openPMD-api via: [conda-forge]
 - operating system: [Ubuntu 18.04.5 LTS]
 - machine: [Not a Public Cluster]
 - name and version of Python implementation: [Python 3.6.9]
 - version of HDF5: [e.g. 1.12.0]
 - version of ADIOS1: [ 1.13.1]
 - version of ADIOS2: [ 2.6.0  ]
 - name and version of MPI: [ (openMPI) 2.1.2  ]

**Additional context**
@PrometheusPi

**Comment by PrometheusPi** [comment]
I could reproduce this locally using `0.12.0-alpha` and `0.8.0-alpha`. 

`series.set_machine()` appears to not exist.

**Comment by ax3l** [comment]
Thanks for the report and proposed PR to expose this functionality in Python #796.
This indeed slipped, let's add this.

The temporary work-around for this is:
```py
series.set_attribute("machine", "machine-site")
```

**Comment by ax3l** [comment]
fixed with #796
